### ISSUE TITLE
Variant vs VariantSet as subject

### ISSUE THREAD

**Issue by nlwashington** [cmungall/schemas#3]
The current G2P schema has a VariantSet as the subject of the annotation.  
According to the variants.avdl, a variant is part of a variantSet, but not the other way around.  As in, it doesn't seem that a VariantSet has Variant members; rather a Variant is only part of a single VariantSet.  (is this odd in and of itself?  since a Variant could be part of many VariantSets, i don't understand this relationship.)

VariantSet is defined as "equal to a VCF file".

However, I think either a Variant or a VariantSet could be subject of a G2P association (especially when whole genomes haven't been sequenced).

**Comment by cmungall** [comment]
Yes, I think it was a mistake for us to reference a VariantSet here, which is essentially a VCF file.

We may have need of a Genotype object, which is, minimally, a collection of variants (and may in many cases be a singleton)
